bin/update_pods.py

```diff
@@ -157,6 +157,24 @@ def _needs_update(repository_ctx):
     cached_version = _exec(repository_ctx, ["/bin/bash", "-c", "cat .pod-version"], pod_root_dir).split("\n")[0]
     return GetVersion(repository_ctx) != cached_version
 
+def _load_repo_if_needed(repository_ctx, repo_tool_bin_path):
+    url = repository_ctx.url
+    target_name = repository_ctx.target_name
+    if not url:
+        # We allow putting source code in the Vendor/PodName and then initing
+        # the repo with that code.
+	return
+
+    # Note: the pod is not cleaned out if the sourcecode is loaded from the
+    # current directory
+    _exec(repository_ctx, ["rm", "-rf", repository_ctx.GetPodRootDir()])
+    _exec(repository_ctx, ["mkdir", "-p", repository_ctx.GetPodRootDir()])
+
+    if url.startswith("http") or url.startswith("https"):
+        _fetch_remote_repo(repository_ctx, repo_tool_bin_path, target_name, url)
+    elif url.startswith("/"):
+        _link_local_repo(repository_ctx, target_name, url)
```

Comment:
There might be better APIs to determine if a url is file or remote

Reply:
I'm not familiar with such an API but that would bee pretty sweet. The nuance here is we either support absolute paths ( of for temporary local dev ) or remote URL's.